terms.html

```diff
@@ -15,20 +15,21 @@
 A set of one or more <a>claims</a> made by the issuer.
 A <dfn data-lt="verifiable credentials">verifiable credential</dfn>
 is a credential that is tamper-resistant and whose authorship can be
-cryptographically verified.
+cryptographically verified. Credentials can be used to build
+<a>presentations</a>, which can also be cryptographically verified.
```

Comment:
We need to be careful with the language here. A credential and a presentation are not cryptographically verifiable. A **verifiable credential** and a **verifiable presentation** are cryptographically verifiable.

Reply:
Will adjust.